README.md

```diff
@@ -227,6 +227,221 @@ For better development experience: the files in your _HOST_ folder `MyAEGEE` are
+    - configure a name and email address you're willing to disclose in the history. GitHub offers you the possibility to use an anonymous address. [This article](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address) explains both possibilities.
+
+
+### Local repositories' organization on the filesystem
+
+The `MyAEGEE` repository contains several git submodules. Submodules are roughly pointers to specific versions of other git repositories. You will most probably **not** commit to the current repository, but you will rather make edits on the frontend, events, or discounts repositories.
+
+```bash
+# steps from the readme
+git clone --recursive git@github.com:AEGEE/MyAEGEE.git
+cd MyAEGEE
+# do a second clone of the repos that are of interest
+mkdir myRepoClones
+cd myRepoClones
+git clone git@github.com:AEGEE/core.git
+git clone git@github.com:AEGEE/events.git
+git clone git@github.com:AEGEE/frontend.git
+git clone git@github.com:AEGEE/statutory.git
+# so that git does not complain about changes when using git from the "MyAegee" folder.
+echo "**" > .gitignore
+```
+
+The folder structure you should have after running the steps above looks like this (the output below was trimmed a bit)
+```
+user@computer:~/Documents/aegee/MyAEGEE$ tree -L 2 -a
+.
+├── base-docker-compose.yml
+├── .circleci
+│   └── config.yml
+├── core
+│   ├── CHANGELOG.md
+│   └── .yamllint.yml
+├── .env
+├── .env.example
+├── events
+│   ├── apiary.apib
+│   └── .yamllint.yml
+├── helper.sh
+├── Makefile
+├── monitor
+│   └── docker
+├── myRepoClones
+│   ├── core
+│   ├── discounts
+│   ├── events
+│   ├── frontend
+│   ├── .gitignore
+│   └── statutory
+├── oms-global
+(...)
+```
+
+Edit the `.env` file and add the second section below the "Paths in the filesystem"
+```yaml
+# Paths in the filesystem
+PATH_GATEWAYS=gateways/docker/
+PATH_CORE=core/docker/
+PATH_EVENTS=events/docker/
+PATH_STATUTORY=statutory/docker
+PATH_FRONTEND=frontend/docker/
+PATH_DISCOUNTS=discounts/docker/
+PATH_MAILER=mailer/docker/
+PATH_PASS_MANAGER=pass-manager/docker/
+PATH_MONITOR=monitor/docker/
+PATH_GSUITE_WRAPPER=gsuite-wrapper/docker/
+PATH_OMS_ELASTIC=oms-elastic/docker/
+
+# My private repos, overriding the default paths
+PATH_EVENTS=myRepoClones/events/docker/
+PATH_FRONTEND=myRepoClones/frontend/docker/
+#PATH_STATUTORY=myRepoClones/statutory/docker
+```
+
+With this setup, you can easily override the code running in production (= the code in the submodules) with the one you have locally (the repos in `myRepoClones`). Having duplicated keys in a `.env` will be displayed as a warning in your IDE, so you will notice through IDE warnings when you are overriding repositories and when you are using the version in production. In the example above, I'm using my own version of events and frontend, but the prod version of statutory.
```

Comment:
Why would you want to use this? I think it is easier to just use the code from the submodules (but pulled to master ofc)